typedoc.js

```diff
@@ -2,15 +2,16 @@ const fs = require("fs")
 const path = require("path")
 
 const EXCLUDE_PKGS = [
+    "circuits",
+    "cli",
     "cli-template",
     "cli-template-contracts-hardhat",
     "cli-template-monorepo-ethers",
     "cli-template-monorepo-subgraph",
-    "core",
-    "circuits",
     "contracts",
+    "core",
     "hardhat",
-    "cli"
+    "heyauthn"
 ]
```

Comment:
Hey @sripwoud 

I think we can remove `heyauthn` since it's not part of this repository anymore. It's now part of the Semaphore extensions repository: https://github.com/semaphore-protocol/extensions

Reply:
https://github.com/semaphore-protocol/semaphore/pull/790/commits/6d87016423a1ca7f316944141f24f992ae71563d